db/sqlglue.c

```diff
@@ -12518,7 +12518,8 @@ int comdb2_check_vtab_access(sqlite3 *db, sqlite3_module *module)
             int bdberr;
             int rc;
 
-            if (module->access_flag == CDB2_ALLOW_ALL) {
+            if ((module->access_flag == 0) ||
```

Comment:
I guess you need `access_flags == 0` here because some of system tables don't set their `access_flag`?

Reply:
Right